opensubtitles/authentication.go

```diff
@@ -0,0 +1,40 @@
+package opensubtitles
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// AuthenticationService provides access to the login related functions
+// in the OpenSubtitles API.
+//
+// OpenSubtitles API docs: https://www.opensubtitles.com/docs/api/html/index.htm#authentication
+type AuthenticationService service
+
+type LoggedIn struct {
```

Comment:
Codacy found an issue: [exported type LoggedIn should have comment or be unexported](https://app.codacy.com/manual/TheForgotten69/go-opensubtitles/pullRequest?prid=6093402)